Dockerfile

```diff
@@ -0,0 +1,62 @@
+FROM redis:3.2 AS redis
+
+FROM traefik:1.4.4 AS traefik
+
+FROM node:8.4.0 AS node
+
+FROM ruby:2.4.2
```

Comment:
This is different than our .ruby-version file, should be bump that file too?

Reply:
yes, but not with this PR. I want to bump the node version as well.